Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/source/StatisticsProvider.kt

```diff
@@ -78,6 +83,9 @@ class StatisticsProvider @Inject constructor(
         val rawData = server.getRawStatistics()
         return parser.parse(rawData).also {
             Timber.tag(TAG).d("Parsed from server: %s", it)
+            if (!it.isDataAvailable) {
+                Timber.tag(TAG).e("RawData: ${rawData.toHexString()}")
```

Comment:
Create an func for that, so you don't have to repeat yourself
Prefer to use %s instead of string templates when using Timber